pkg/monitortests/testframework/uploadtolokiserializer/monitortest.go

```diff
@@ -249,8 +249,13 @@ func UploadIntervalsToLoki(intervals monitorapi.Intervals) error {
 			labels["namespace"] = logLine["namespace"]
 		}
 
+		// When 'intervals' was processed by Intervals.Slice(), the value of Interval.To is
+		// used to tell if the interval belongs in the slice. Since the value of Interval.To
+		// is always more recent than (or equal to) Interval.From, we use it for the
+		// timestamp here to reduce the chances of getting the 'entry too far behind' error
+		// from Loki.
 		values = append(values, []interface{}{
-			strconv.FormatInt(i.From.UnixNano(), 10),
+			strconv.FormatInt(i.To.UnixNano(), 10),
```

Comment:
This doesn't look right, this looks like the place we're defining the timestamp at which the "log" line took place, that is the from for an interval, if we change it to the "to", then ordering will go wonky when we query, things that started before other things will appear out of order depending on when the interval finished. from/to should not be the cause of the too old problems in loki afaict.

Reply:
hm... ok will put that back.